app/uk.gov.hmrc.helptosavestridefrontend/views/partials/h2.scala.html

```diff
@@ -0,0 +1,33 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@this()
+
+@(
+  key: String,
+  classes: Option[String] = None
+)(implicit
+  messages: Messages
+)
+
+@cls = @{"govuk-heading-s".concat(classes.getOrElse(""))}
+
+<h2 class="@cls">@Html(messages(key))</h2>
```

Comment:
I don't think we should use indiscriminate HTML wrapping (i.e. wrapping ANY value as if it were safe HTML) for message output.

Reply:
done